zerberus/sfz.cpp

```diff
@@ -266,63 +285,76 @@ static void readInt(const QString& data, int* val)
 
 void SfzRegion::readOp(const QString& b, const QString& data)
       {
-      int i = data.toInt();
+      QStringList splitData = data.split(" ");
```

Comment:
Could you add a comment here.
*No space in opcode value except for sample*
Or something like that.